### PR TITLE
update README

### PR DESCRIPTION
**Pull request by Pahandrovich**
Changes proposed in this pull request:
- update info about patching in readme and update links in docs

### CODE REVIEW

**Review by averbukh — APPROVED** [review]

**Review by napetrov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -82,30 +82,18 @@ result = alg.compute(data, centroids)
 
 ### Scikit-learn patching
 
-With a Python API provided by daal4py, you can create scikit-learn compatible estimators, transformers, or clusterers that are powered by oneDAL and are nearly as efficient as native programs.
+You can speed up Scikit-learn using Intel(R) Extension for Scikit-learn*.
+
+Intel(R) Extension for Scikit-learn* speeds up scikit-learn beyond  by providing drop-in patching. Acceleration is achieved through the use of the Intel(R) oneAPI Data Analytics Library ([oneDAL](https://github.com/oneapi-src/oneDAL)) that allows for fast usage of the framework suited for Data Scientists or Machine Learning users.
 
 | *Speedup of oneDAL-powered scikit-learn over the original scikit-learn, 28 cores, 1 thread/core* |
```
Comment (SmirnovEgorRu) [inline README.md:89]: Rename the chart name

File: README.md [hunk 2]
```diff
@@ -82,30 +82,18 @@ result = alg.compute(data, centroids)
 
 ### Scikit-learn patching
```
Comment (SmirnovEgorRu) [inline README.md:83]: I think it should be ##

File: README.md [hunk 3]
```diff
@@ -82,30 +82,18 @@ result = alg.compute(data, centroids)
...
 
-With a Python API provided by daal4py, you can create scikit-learn compatible estimators, transformers, or clusterers that are powered by oneDAL and are nearly as efficient as native programs.
+You can speed up Scikit-learn using Intel(R) Extension for Scikit-learn*.
+
+Intel(R) Extension for Scikit-learn* speeds up scikit-learn beyond  by providing drop-in patching. Acceleration is achieved through the use of the Intel(R) oneAPI Data Analytics Library ([oneDAL](https://github.com/oneapi-src/oneDAL)) that allows for fast usage of the framework suited for Data Scientists or Machine Learning users.
```
Comment (SmirnovEgorRu) [inline README.md:87]: Need to add a link to skex and remove the link to oneDAL.

File: README.md [hunk 4]
```diff
@@ -124,17 +112,17 @@ end = time()
...
 start = time()
-svm_d4p = svm_d4p.fit(X, y)
+svm_sklearnex = svm_sklearnex.fit(X, y)
 end = time()
 print(end - start) # output: 0.032536...
-print(svm_d4p.score(X, y)) # output: 0.9905397885364496
+print(svm_sklearnex.score(X, y)) # output: 0.9905397885364496
 ```
 
 ### Distributed multi-node mode
```
Comment (SmirnovEgorRu) [inline README.md:128]: Move to Python API part

File: README.md (hunk 3 above)
Comment (SmirnovEgorRu) [reply README.md:87]: + link to docs

**Review by SmirnovEgorRu — APPROVED** [review]

**Pull request merged**